source/dub/internal/vibecompat/core/file.d

```diff
@@ -145,8 +145,14 @@ version (Windows) extern(Windows) int CreateHardLinkW(in wchar* to, in wchar* fr
 // guess whether 2 files are identical, ignores filename and content
 private bool sameFile(Path a, Path b)
 {
-	static assert(__traits(allMembers, FileInfo)[0] == "name");
-	return getFileInfo(a).tupleof[1 .. $] == getFileInfo(b).tupleof[1 .. $];
+	version (Posix) {
+		auto st_a = std.file.DirEntry(a.toNativeString).statBuf;
+		auto st_b = std.file.DirEntry(a.toNativeString).statBuf;
+		return st_a == st_b;
```

Comment:
This does an identity check, but an identical copy would be OK as well.
How about just using the same getFileInfo heuristic?